pages/docs/reference/data-classes.md

```diff
@@ -119,6 +119,28 @@ val olderJack = jack.copy(age = 2)
 
 </div>
 
+Note that the `copy()` call performs shallow copy, meaning that it doesn't propagate recursively through attributes:
```

Comment:
I suggest adding a clearer description of how `copy` treats reference and primitive properties.
```suggestion
Note that the `copy()` call performs a shallow copy, meaning that the copy's reference properties point to the _same objects_ as the original object's references. However, the copy has its own primitive properties not linked to the original.
```